test/conformance/program/urProgramCreateWithBinary.cpp

```diff
@@ -10,11 +10,14 @@ struct urProgramCreateWithBinaryTest : uur::urProgramTest {
         UUR_RETURN_ON_FATAL_FAILURE(urProgramTest::SetUp());
         ASSERT_SUCCESS(urProgramBuild(context, program, nullptr));
         size_t binary_size = 0;
-        ASSERT_SUCCESS(urProgramGetInfo(program, UR_PROGRAM_INFO_BINARIES, 0,
-                                        nullptr, &binary_size));
+        ASSERT_SUCCESS(urProgramGetInfo(program, UR_PROGRAM_INFO_BINARY_SIZES,
```

Comment:
I guess this should be returning a a `size_t[]` so we maybe could expand the test by making sure the returned array size is only one, but not sure if that is adding value tbh, so feel free to ignore that.

Reply:
good shout, added a check before this that just queries the size and makes sure it's in line with the `size_t[1]` we're expecting